### ISSUE TITLE
About fine-tuned code and data sets

### ISSUE THREAD

**Issue by potentialming** [littlewhitesea/StitchDiffusion#4]
Hello, what a great idea. I learned from the paper that you used Panorama to fine-tune it. Could you share the fine-tuning code and data set? I would be very grateful.

**Comment by littlewhitesea** [comment]
Hi, thanks for your interest in our work. We have provided the fine-tuned [dataset](https://drive.google.com/file/d/1EgRwj5BqO7Y-PvdL8mrFwKsqmgN_N4_b/view) in this repository. As for the fine-tuning code, there are many excellent open-sourced codes in the github to train a LoRA for the latent diffusion model.

**Comment by Ir1d** [comment]
Hi @littlewhitesea thanks for the great work. During training, what resolution do you use for calculating the l_pano loss? 
![image](https://github.com/littlewhitesea/StitchDiffusion/assets/10709657/db84f0b3-c924-40bb-af46-c93ceba1306f)
Do you use the whole panorama image directly in each training iteration?

**Comment by littlewhitesea** [comment]
Thanks for your interest. Yes, we use the whole panorama image with dimensions 512x1024 directly during the fine-tuning process.
